cmd/okctl/hooks/state.go

```diff
@@ -230,6 +230,28 @@ func PurgeRemoteState(o *okctl.Okctl) RunEer {
 	}
 }
 
+// VerifyClusterExistsInState ensures we have a cluster in the state before continuing
+// This could happen if maintenance mode was not run after upgrading
+// to 0.0.80 and we don't have a state.db in S3 - see #491
+func VerifyClusterExistsInState(o *okctl.Okctl) RunEer {
+	return func(_ *cobra.Command, _ []string) error {
+		handlers := o.StateHandlers(o.StateNodes())
+
+		_, err := handlers.Cluster.GetCluster(o.Declaration.Metadata.Name)
+		if err != nil {
+			o.Logger.Debug("Could not find cluster in current state: ", err)
```

Comment:
I'd suggest using our new `pkg/logging` package. It's going to be our standard way of logging going forward

Initialising the logger requires the string arguments "component" and "activity", in this case I'd say the component is "hooks" or "stateHook", and the activity is "VerifyClusterExistsInState"